libs/solid/skeleton/src/lib/components/feedback/feedback.component.html

```diff
@@ -1,5 +1,5 @@
-<h1 mat-dialog-title>{{ formTitle }}</h1>
-<div mat-dialog-content>
+<h1 style="font-weight: bold" mat-dialog-title>{{ formTitle }}</h1>
```

Comment:
Erinnerung: keine inline styles! Bitte auch in so einem Fall eine `class` vergeben und die *.scss verwenden

Und ich finde das Default-Styling (font-weight 500) für die Überschrift schon gut, das muss nicht bold sein.
(Zumindest bei mir auf Windows & Chrome ist das Rendering von Roboto bold nicht schön.)